tests/sample_eventgen_conf/unit/eventgen.conf.config

```diff
@@ -0,0 +1,38 @@
+[sample]
+sampleDir = tests/sample_eventgen_conf/sample
+outputMode = splunkstream
```

Comment:
can we start with HECendpoint and modinput outputmode?

Reply:
This config is not actually used to stream data into Splunk. It is used only for the config unit test.